app/views/users/index.html.erb

```diff
@@ -10,52 +10,61 @@
 
 <div class="row">
   <% if policy(User).manage? %>
-    <div class="col-xs-12 col-sm-6">
-      <h2><%= t('add new member') %></h2>
-      <div class="add-member">
-        <%= form_for @user,
-          url: project_users_path(@project, @user),
-          html: { class: 'form-horizontal' } do |f| %>
+    <div class="col-xs12 col-xs-8 col-sm-offset-2">
+      <div class="page-header">
+        <h4 class="page-header-title darkgrey-2">
+          <i class="mi md-20">dashboard</i> <%= t('add new member') %>
+        </h4>
+      </div>
+    </div>
+
+    <div class="col-xs-12 col-sm-4 col-sm-offset-2">
+
+      <div class="add-member panel panel-default card">
+        <div class="panel-body">
+          <%= form_for @user,
+            url: project_users_path(@project, @user) do |f| %>
 
-          <%= content_tag :div, class: 'alert alert-danger' do %>
-            <%= t('errors.template.header', count: @user.errors.count, model: @user.class.name.humanize) %>
+            <%= content_tag :div, class: 'alert alert-danger' do %>
+              <%= t('errors.template.header', count: @user.errors.count, model: @user.class.name.humanize) %>
 
-            <ul>
-              <% @user.errors.full_messages.each do |msg| %>
-                <li><%= msg %></li>
-              <% end %>
-            </ul>
-          <% end if @user.errors.any? %>
+              <ul>
+                <% @user.errors.full_messages.each do |msg| %>
+                  <li><%= msg %></li>
+                <% end %>
+              </ul>
+            <% end if @user.errors.any? %>
 
-          <div class="form-group">
-            <%= f.label :email, class: 'col-xs-2 control-label' %>
-            <div class="col-xs-10"><%= f.text_field :email, class: 'form-control' %></div>
-          </div>
+            <div class="form-group">
+              <%= f.label :email, class: 'control-label' %>
+              <%= f.email_field :email, class: 'form-control auth-form-control', :required => true %>
```

Comment:
You could use the new ruby hash syntax: `required: true`. It's just a suggestion